packages/vats/tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "*.js",
     "scripts/**/*.js",
     "src/**/*.js",
+    "src/**/*.ts",
```

Comment:
Should this be

```suggestion
    "src/**/*.d.ts",
```

Reply:
What's here works, but would allow `src` to have runtime `.ts` files so I think your suggestion is worth including. I'll do it in a cleanup push